src/cubicbez.rs

```diff
@@ -256,13 +256,13 @@ impl CubicBez {
         (a, b, c, d)
     }
 
+    /// Rust port of cu2qu [calc_cubic_points](https://github.com/fonttools/fonttools/blob/3b9a73ff8379ab49d3ce35aaaaf04b3a7d9d1655/Lib/fontTools/cu2qu/cu2qu.py#L63-L68)
     fn from_parameters(a: Vec2, b: Vec2, c: Vec2, d: Vec2) -> Self {
-        CubicBez::new(
-            d.to_point(),
-            d.to_point() + (c / 3.0),
-            d.to_point() + (c / 3.0) + (b + c) / 3.0,
-            d.to_point() + (c / 3.0) + (b + c) / 3.0 + a,
```

Comment:
good catch! it was only this line that computed the CubicBez end point `p3` that was at fault (the other three points were correct, you've simply rewritten to match fonttools' original code more literally, which is good). p3 should have been the sum of the four parameters a + b + c + d instead.

Reply:
Correct, I attempted to highlight that in the PR description but perhaps I should have stuck a comment here too.

Reply:
> "was computing the final point differently"

Oh I now reread that and makes sense, thank you